docs(firestore-bigquert-export): Updated documentation for gen-schema column overides (#597)
firebase/extensions: firestore-bigquery-export/guides/GENERATE_SCHEMA_VIEWS.md

}
```

> Biq Query has a number of reserved keywords that can cause conflicts when creating a schema, `timestamp` is one such example. To ensure column names stay consistent when syncing with Firestore, this can overridden using the `column_name` option when defining a field.

Reviewer: ```suggestion
Biq Query has a number of reserved keywords that can cause conflicts when creating a schema, `timestamp` is one such example. To ensure column names stay consistent when syncing with Firestore, they can overridden using the `column_name` option when defining a field.
```

> Biq Query has a number of reserved keywords that can cause conflicts when creating a schema, `timestamp` is one such example. To ensure column names stay consistent when syncing with Firestore, they can be overridden using the `column_name` option when defining a field.

Reviewer 2: ```suggestion
SQL has a number of [reserved keywords](https://en.wikipedia.org/wiki/SQL_reserved_words) that can cause conflicts when creating a schema, `timestamp` is one such example. To ensure your Firestore document field names do not conflict, use the `column_name` option to override the field name.
```

Please see the example below...

```
{
  "fields": [
    {
      "name": "name",
      "type": "string"
    },
    {
      "name": "age",

>       "type": "number"

Reviewer: ```suggestion
      "type": "number",
```

>       "column_name": "new_column_name" //Can be used to override column names

Reviewer: ```suggestion
      "column_name": "new_column_name" // Can be used to override column names.
```

>       "column_name": "new_column_name" // Can be used to override column names.

Reviewer 2: ```suggestion
      "column_name": "new_column_name" 
```
